melodic/distribution.yaml

```diff
@@ -1841,13 +1841,17 @@ repositories:
       version: 0.1.12-0
     status: unmaintained
   core_perception:
+    doc:
+      type: git
+      url: https://github.com/nobleo/core_perception.git
+      version: points_preprocessor_release_noetic
```

Comment:
It's a little weird (but allowed) to have the branch here disagree with the branch for the source.  Did you mean for this to be `points_preprocessor_release`?

Reply:
Good catch! Fixed it.